css/pages/authentication.css

```diff
@@ -0,0 +1,67 @@
+/* 
+ *   Authentication form
+*/
+
+.auth-form {
+  background: white;
+  box-shadow: var(--box-shadow-card);
+  padding: 2rem;
+}
+
+.auth-form .form-control-wrapper {
+  margin: 1rem 0;
+}
+
+.auth-form .link {
+  text-decoration: none;
+}
+
+.auth-form .btn:hover {
+  box-shadow: none;
+}
+
+.auth-form .link:hover {
+  font-weight: var(--fw-bold);
+}
+
+/* 
+ *  Logout
+*/
+
+.logout-container {
+  height: 70vh;
+}
+
+.card-logout .icon {
+  font-size: 3rem;
+  color: var(--clr-primary);
+}
+
+.card-logout {
+  max-width: 40rem;
+  background-color: white;
+  padding: 2rem 1rem;
+  box-shadow: var(--box-shadow-card);
+}
+
+.card-logout .logout-msg {
+  margin: 2rem 1rem 1rem 1rem;
+  font-size: 1.25rem;
+}
+
+.card-logout .link {
+  text-decoration: none;
+  border-bottom: 1px solid var(--clr-primary);
+}
+
+.card-logout .link:hover,
+.card-logout .link:hover {
```

Comment:
Any specific reason to use the pseudo element on the same class twice?

Reply:
Thanks for pointing it out! It was by mistake.